elements/lisk-chain/src/chain.ts

```diff
@@ -167,6 +168,7 @@ export class Chain {
 			},
 		};
 		codec.addSchema(accountSchema as Schema);
```

Comment:
I think you can remove this typecasting `as Schema`

Reply:
also at line https://github.com/LiskHQ/lisk-sdk/pull/5415/files#diff-d84e8ad8c9682409e75b789bc2b45f95R178